articles/azure-app-configuration/overview-managed-identity.md

```diff
@@ -56,6 +56,9 @@ The following steps will walk you through creating an App Configuration store an
 
 Creating an App Configuration store with a user-assigned identity requires that you create the identity and then assign its resource identifier to your store.
 
+> [!NOTE]
+> You can add up to 10 user assigned managed identities to your app configuration.
```

Comment:
Thanks for putting together a PR, @rapopescu16. Small suggestion:

... user-assigned managed ... to an App Configuration store.